apps/papra-client/src/modules/organizations/pages/organizations.page.tsx

```diff
@@ -24,40 +63,117 @@ export const OrganizationsPage: Component = () => {
   ));
 
   return (
-    <div class="p-6 mt-4 pb-32 max-w-5xl mx-auto">
-      <h2 class="text-xl font-bold mb-2">
-        {t('organizations.list.title')}
-      </h2>
+    <div class="p-6 mt-4 pb-32 max-w-6xl mx-auto">
+      <div class="mb-8">
+        <h2 class="text-2xl font-bold mb-2">
+          {t('organizations.list.title')}
+        </h2>
 
-      <p class="text-muted-foreground mb-6">
-        {t('organizations.list.description')}
-      </p>
+        <p class="text-muted-foreground">
+          {t('organizations.list.description')}
+        </p>
+      </div>
 
-      <div class="grid gap-4 grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4">
+      <div class="grid gap-6 grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 auto-rows-fr">
         <For each={query.data?.organizations}>
-          {organization => (
-            <A
-              href={`/organizations/${organization.id}`}
-              class="border rounded-lg overflow-hidden"
-            >
-              <div class="bg-card border-b flex items-center justify-center p-6">
-                <div class="size-16 text-muted-foreground"></div>
-              </div>
+          {organization => {
+            const stats = getOrganizationStats(organization.id);
+            return (
+              <A
+                href={`/organizations/${organization.id}`}
+                class="group border rounded-xl overflow-hidden bg-card hover:shadow-lg transition-all duration-200 hover:border-primary/20 hover:-translate-y-1"
+              >
+                <div class="p-6 h-full flex flex-col">
+                  {/* Organization Icon and Header */}
+                  <div class="mb-4">
+                    <div class="flex items-start gap-4 mb-2">
```

Comment:
What about:

```suggestion
                    <div class="flex items-center gap-4 mb-2">
```

In order to align the icon and the name ?